test/srt/test_serving_latency.py

```diff
@@ -12,7 +12,7 @@ def test_default(self):
             "python3",
             "-m",
             "sglang.bench_latency",
-            "--model",
+            "--model-path",
```

Comment:
`--model` could be `--model-path` or `--model-override-args`.

Reply:
It's better to keep the original while you can add others.

Reply:
Can you rename `--model-override-args` to `--override-model-args`?

Reply:
> Can you rename --model-override-args to --override-model-args?

How about `--json-model-override-args`, or do you prefer `--override-model-args`?

> In the meanwhile, we can change all --model in the code base to --model-path to avoid future ambiguity

<img width="855" alt="image" src="https://github.com/user-attachments/assets/a2adc0a7-2c05-4a39-bc5d-44422bfcd339">